usl_pipeline/usl_lib/tests/storage/test_metastore.py

```diff
@@ -242,6 +242,47 @@ def test_flood_scenario_config_delete():
     )
 
 
+def test_heat_scenario_config_set():
```

Comment:
I'd generally advocate for writing integration tests against a local firestore emulator for more realistic tests, such as you see in here:
https://github.com/UrbanSystemsLab/climateiq-cnn/blob/main/usl_pipeline/usl_lib/tests/integration/test_metastore_integration.py#L34